README.md

```diff
@@ -7,22 +7,22 @@
 [![Coderwall Endorse](http://api.coderwall.com/flyerhzm/endorsecount.png)](http://coderwall.com/flyerhzm)
 [![Click here to lend your support to: rails best practices and make a donation at www.pledgie.com !](https://pledgie.com/campaigns/12057.png?skin_name=chrome)](https://pledgie.com/campaigns/12057)
 
-rails_best_practices is a code metric tool to check the quality of rails code.
+rails_best_practices is a code metric tool to check the quality of Rails code.
 
-it supports following ORM/ODMs:
+It supports the following ORM/ODMs:
 
 * activerecord
 * mongoid
 * mongomapper
 
-following template engines:
+And the following template engines:
 
 * erb
 * haml
 * slim
 * rabl
 
-rails_best_practices works well only in ruby 1.9.3, 2.0.0, and 2.1.0 so far.  It is incompatible with ruby 1.8.
+rails_best_practices supports Ruby 1.9.3 or newer.
```

Comment:
Mentioning 1.8 explicitly seems redundant. And since Ruby versions are generally backwards compatible, it doesn't seem necessary to list each release individually.